src/test/java/teammates/test/cases/browsertests/InstructorCourseEnrollPageUiTest.java

```diff
@@ -134,8 +134,8 @@ private void testEnrollAction() throws Exception {
             .withUserId(testData.instructors.get("CCEnrollUiT.teammates.test").googleId)
             .withCourseId(courseId);
         detailsPage = loginAdminToPage(coursesPageUrl, InstructorCoursesDetailsPage.class);
-        assertEquals(3, detailsPage.getStudentCountForCourse("CCEnrollUiT.CS2104"));
-
+        assertEquals(3, detailsPage.getStudentCountForCourse());
+      
```

Comment:
I will certainly try to be more careful. Going through the process.md , the merging is to be done by dev(with push permission) or reviewer. 
So I think my job is done, or is anything left?